### ISSUE TITLE
Google Maps images are failing for events

### ISSUE THREAD

**Issue by stof** [afsy/website#164]
The server returns a 403 with this content:

> The Google Maps Platform server rejected your request. You must use an API key to authenticate each request to Google Maps Platform APIs. For additional information, please refer to http://g.co/dev/maps-no-account

**Comment by qdequippe** [comment]
Did you think about change map provider? maybe [Mapbox](https://www.mapbox.com/pricing/) is less expensive than Google Maps

**Comment by clem** [comment]
Yes. Or maybe we can change for [OpenStreetMap](https://www.openstreetmap.org/) 😃

**Comment by qdequippe** [comment]
@clem you're right OpenStreetMap is event better!

**Comment by Nek-** [comment]
For your information, we were not really using GMaps API. It's just an image link to gmaps hand-written.

It's cool because it allows us to use another image (for example christmas one for avent calendar).

It's not cool because it's obviously a pain to write 😬.

If OSM has some similar features, that would be awesome.

**Comment by stof** [comment]
Well, this image link is part of the GMaps API. Look at the website. None of these images are working.

**Comment by qdequippe** [comment]
@Nek- I checked quickly and OSM does not provide Static render like this :/

**Comment by Nek-** [comment]
@qdequippe I assume we should use something more complex... PR open x)
